kpatch-build/kpatch-build

```diff
@@ -140,12 +153,54 @@ gcc_version_check() {
 		return 1
 	fi
 
-	# ensure gcc version is >= 4.8
-	gccver=$(echo $gccver |cut -d'.' -f1,2)
-	if [[ $gccver < 4.8 ]]; then
-		warn "gcc >= 4.8 required"
-		return 1
+	if [[ $ARCH = "ppc64le" ]]; then
+		# On ppc64le livepatch requires -mprofile-kernel support.
+		# Adopt <linux-sources>/arch/powerpc/scripts/gcc-check-mprofile-kernel.sh
+		# for gcc support checks.
```

Comment:
Maybe this comment should be moved to the header of the gcc_mprofile_test() function since it describes its implementation.